### DISCUSSION TITLE
Is it possible to create Realistic animation?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3082] [Q&A] [answered]
Would it perhaps be able to create a short video animation from a photo. We could perhaps convert an Ai generated photo of them into a short video animation.
Goal: give users a short video animation of themselves based on the Ai photos we generated of them.
Perhaps this is a more entertaining feature.

It need not be a full video or very long. Simply some movement / animation. To catch the eye and delight.

**Comment by tommy3210** [comment]
Well i think it's possible.  A few days ago, when I was scrolling through Twitter, I found this.  I haven't tried it yet but I think it can be done with reality images

**Reply by oldhand7** [reply]
Thank you. 
I want to 
- upload selfi photo
- input prompt like "flying in the sky"

- output: the uploaded user flying in the sky in 5 - 6 s.
Would it be possible?
If so, please let me know how to do.
THank you.

**Reply by mashb1t** [reply, marked as answer]
Animation / video generation is not part of Fooocus features.
There are plenty of tools out there where you can start with a selfie and define an animation. If you want to use Open Source tools you can check out ComfyUI and [AnimateDiff](https://animatediff.github.io/), [Stable Video Diffusion](https://stability.ai/stable-video) or similar, also see https://stable-diffusion-art.com/animatediff/.

**Reply by oldhand7** [reply]
Thank you @mashb1t , 
Can I create the video of selfi photo users upload with AnimateDiff?

**Reply by mashb1t** [reply]
Yes. https://github.com/guoyww/animatediff/?tab=readme-ov-file#quick-demos
